store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreVisibilityTest.java

```diff
@@ -52,17 +48,19 @@
 import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.STRING;
 import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.loadAccumuloStoreProperties;
 import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.resetForFederatedTests;
+import static uk.gov.gchq.gaffer.store.TestTypes.INTEGER_TYPE;
+import static uk.gov.gchq.gaffer.store.TestTypes.STRING_TYPE;
+import static uk.gov.gchq.gaffer.store.TestTypes.VISIBILITY;
+import static uk.gov.gchq.gaffer.store.TestTypes.VISIBILITY_2;
 
 public class FederatedStoreVisibilityTest {
     private static final AccumuloProperties ACCUMULO_PROPERTIES = loadAccumuloStoreProperties(ACCUMULO_STORE_SINGLE_USE_PROPERTIES);
 
     public static final String PUBLIC = "public";
-    private static final User USER = new User.Builder().dataAuth(PUBLIC).build();
-    public static final String VISIBILITY = "visibility";
-    public static final String VISIBILITY_1 = "visibility1";
-    public static final String VISIBILITY_2 = "visibility2";
     public static final String PRIVATE = "private";
 
+    private static final User USER = new User.Builder().dataAuth(PUBLIC).build();
```

Comment:
The two which are public can simply be private as they are not used outside this file. The line breaks between these fields can then be removed entirely.